.github/workflows/build.yml

```diff
@@ -5,6 +5,8 @@ on:
       - master
     paths-ignore:
       - "**/README.md"
+permissions:
+   contents: write
```

Comment:
What are you doing?

Reply:
i see that you do not want to merge this so i tried to build my own .dat file it needs the permission to release. I forgot that i didn't create a new branch. if you want to merge i will revert it :D

Reply:
i have remove those changes

Reply:
You can see README.md.
"""
A script will verify your pull request to test whether your PR is correct or not every time you update the PR. Only the PR which passes the test will be merged. Please go to the Action label to get detailed information if you didn't pass it. We also provide the file which has been generated to make you test.
"""
So you can get the file in GitHub action.